packages/lib-classifier/src/components/Classifier/components/TaskArea/components/DisabledTaskPopup/DisabledTaskPopup.js

```diff
@@ -0,0 +1,84 @@
+import { Modal, PlainButton, PrimaryButton } from '@zooniverse/react-components'
+import counterpart from 'counterpart'
+import { Paragraph } from 'grommet'
+import { bool, func } from 'prop-types'
+import React, { useEffect, useState } from 'react'
+
+import en from './locales/en'
+counterpart.registerTranslations('en', en)
+
+/**
+ A popup which interrupts classification for subjects that are retired or already seen.
+
+ Volunteers are offered options to choose something else to work on, or to dismiss the popup and continue anyway.
+
+ ```
+  <DisabledTaskPopup
+   isOpen
+   onClose={callback}
+   target={parentNode}
+  />
+ ```
+*/
+export default function DisabledTaskPopup({
+  isOpen = false,
+  nextAvailable,
+  onClose = () => true,
+  reset,
+  target
+}) {
+  const [ active, setActive ] = useState(isOpen)
+
+  useEffect(function onLoaded() {
+    if (isOpen) {
+      setActive(true)
+    }
+  }, [isOpen])
+
+  function closeModal() {
+    setActive(false)
+    onClose()
+  }
+
+  return (
+    <Modal
+      active={active}
+      full='horizontal'
+      modal={false}
+      position='top'
+      target={target}
+      title={counterpart('DisabledTaskPopup.title')}
+    >
+      <Paragraph>
+        {counterpart('DisabledTaskPopup.body')}
+      </Paragraph>
+      <PlainButton
+        alignSelf="center"
+        onClick={reset}
+        text={counterpart('DisabledTaskPopup.options.select')}
+      />
```

Comment:
'Select a new subject' only makes sense in the context of a workflow that allows subject selection, so this maybe needs to be wrapped in a check for `workflow.hasIndexedSubjects` or similar.

Reply:
The task area now checks for `workflow.hasIndexedSubjects` before showing this modal. The content is only relevant to Engaging Crowds, at the moment.